src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
+
+function expandTree(treeIn) {
+  for (const item of treeIn.querySelectorAll('li')) {
+    if (item.querySelector('ul')) {
+      expandTree(item.querySelector('ul'));
+
+      const newSpan = document.createElement('span');
+
+      newSpan.textContent = item.firstChild.textContent.trim();
```

Comment:
The line `newSpan.textContent = item.firstChild.textContent.trim();` assumes that the first child of the list item is a text node. If the first child is not a text node, this will cause an error. Consider checking if `item.firstChild` is a text node before accessing `textContent`.